docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: 'Manage Data Assets'
+title: 'Manage Data Assets'
+description: Create and manage Data Assets in GX Cloud.
+---
+
+A Data Asset is a collection of records that you create when you connect to your Data Source. When you connect to your Data Source, you define a minimum of one Data Asset. You use these Data Assets to create the Batch Requests that define the data that is provided to your Expectations.
+
+## Prerequisites
```

Comment:
I don't know where this doc snippet lives. Is it specific for cloud and snowflake? This prerequisites seem to indicate this. One can create in OSS alone and for datasources beside snowflake.

Reply:
Discussed with @billdirks and this can stay as is.